demo/SalesDemoApp/SalesDemoApp/CategoryViewController.m

```diff
@@ -74,6 +74,52 @@ -(void)btnCartPressed:(id)sender{
     ListItemCartViewController *cartVC = [self.storyboard instantiateViewControllerWithIdentifier:@"ListItemCartViewController"];
```

Comment:
we should remove commented code ?

Reply:
Had kept that code as sample for capture item /transaction / Persona , will uncomment this.